modules/rhel-preparing-node.adoc

```diff
@@ -68,7 +68,9 @@ Alternatively, disable all repositories:
 +
 Note that this might take a few minutes if you have a large number of available repositories
 
-. Enable only the repositories required by {product-title} {product-version}:
+. Enable only the repositories required by {product-title} {product-version}.
+
+.. For {op-system-base} 7 nodes, you must enable the following repositories:
```

Comment:
If we deprecated RHEL-7 workers in OCP-4.9, shall we remove the RHEL-7 worker scaling-up section?

Reply:
RHEL 7 support was [deprecated in 4.6](https://docs.openshift.com/container-platform/4.6/release_notes/ocp-4-6-release-notes.html#ocp-4-6-byo-rhel-7-deprecated) but is still supported in 4.9.  RHEL 7 support will be removed in a release after 4.9.

See also 4.7 release notes [deprecated features](https://docs.openshift.com/container-platform/4.7/release_notes/ocp-4-7-release-notes.html#ocp-4-7-deprecated-removed-features).
It looks like RHEL 7 was inadvertently(?) dropped from the [deprecated features table in 4.8](https://docs.openshift.com/container-platform/4.8/release_notes/ocp-4-8-release-notes.html#ocp-4-8-deprecated-removed-features).

Reply:
@gpei as Russell said, since RHEL7 workers are still supported, I was leaving these instructions in. I have added a note below this to state its deprecation. What do you think?

@mtnbikenc I will re-add the RHEL7 deprecation for the 4.8 release notes

Reply:
@codyhoag  It sounds good to me. Thanks for the info from @mtnbikenc .